### ISSUE TITLE
windows 7 iso failed

### ISSUE THREAD

**Issue by kabnaj** [slacka/WoeUSB#91]
windows 7 iso failed on creating directory $partitionMountPath/EFI/Boot as it already existed lowercase.
Command used:
`winusb -v --format W7SP1_PROFESSIONAL_NEW_64.iso /dev/sdc`

This workaround helped:]

```diff
--- ./winusb.old        2017-02-26 19:04:29.000000000 +0100
+++ ./winusb    2017-03-18 16:31:20.330448410 +0100
@@ -390,7 +390,7 @@
 if ( grep -Eq "^MinServer=7[0-9]{3}\.[0-9]" "$isoMountPath/sources/cversion.ini") && [ -f "$isoMountPath/bootmgr.efi" ]; then
        # It's Windows 7 with EFI support (and thus 64-Bit)
        if command -v "7z" >/dev/null 2>&1; then
-               mkdir -p "$partitionMountPath/EFI/Boot"
+               [ -d $partitionMountPath/efi/boot ] || mkdir -p "$partitionMountPath/EFI/Boot"
                7z e -so "$isoMountPath/sources/install.wim" "1/Windows/Boot/EFI/bootmgfw.efi" > "$partitionMountPath/EFI/Boot/bootx64.efi"
        else
                echo "Warning: '7z' utility missing, EFI booting might not work!" >&2
```

MODIFIED: @Lin-Buo-Ren: Enable syntax highlighting

**Comment by Lin-Buo-Ren** [comment]
It seems pretty straight-forward, thanks for reporting!

**Comment by Lin-Buo-Ren** [comment]
Hello @kabnaj and others that are affected by this issue:

This issue should be resolved since WoeUSB v2.0.11, please check if you still can reproduce this issue, thanks!

**Comment by Lin-Buo-Ren** [comment]
Hello @kabnaj , and other that are affected by this issue.

This issue is resolved since WoeUSB v2.0.11, please comment if you still can reproduce the issue, thanks!

**Comment by liangnicho** [comment]
I searched for a long time and I want to buy Windows 7 pro product key and download genuine iso. Finally I saw on Facebook that other people shared the site [www.windows10productkey.com](https://www.windows10productkey.com/) here , I purchased a key at a very favorable price， successfully activated my Windows 7 pro system, I recommend you to try.
